draft-ietf-quic-version-negotiation.md

```diff
@@ -194,92 +215,49 @@ server.
-all the versions from that Version Negotiation packet in order, even if they
-are not supported by the client (even if the versions are reserved). If the
-client has not received a Version Negotiation packet on this connection, this
-field SHALL be 0.
+Server-Sent Other Versions:
 
-Compatible Version Count:
+: When sent by a server, the `Other Versions` field lists all the
+Fully-Supported Versions of this server deployment, see {{server-terms}}. This
+field conveys Fully-Supported Versions instead of Supported Versions to allow
+scenarios where the client's first flight (before receiving VN) reached a
+different server than the client's subsequent flight (after receiving VN).
 
-: A variable-length integer specifying the number of Compatible Version fields
-following it. The client lists all versions that this first flight is
-compatible with in the subsequent Compatible Version fields, ordered by
-descending preference. Note that the version in the Currently Attempted Version
-field MUST be included in the Compatible Version list to allow the client to
-communicate the currently attempted version's preference. Note that this
-preference is only advisory, servers MAY choose to use their own preference
-instead.
-
-~~~
-Server Handshake Version Information {
-  Negotiated Version (32),
-  Supported Version Count (i),
-  Supported Version (32) ...,
-}
-~~~
-{: #fig-server-hvi title="Server Handshake Version Information"}
-
-The content of each field is described below:
-
-Negotiated Version:
-
-: The version that the server chose to use for the connection. This field MUST
-be equal to the value of the Version field in the long header that carries this
-data.
-
-Supported Version Count:
-
-: A variable-length integer specifying the number of Supported Version fields
-following it. The server encodes all versions it supports in the subsequent
-list, ordered by descending preference. Note that the version in the Negotiated
-Version field MUST be included in the Supported Version list to allow the
-server to communicate the negotiated version's preference. Note that this
-preference is only advisory, clients MAY choose to use their own preference
-instead.
-
-Clients MAY include versions following the pattern `0x?a?a?a?a` in their
-`Compatible Version` list, and the server in their `Supported Version` list.
-Those versions are reserved to exercise version negotiation (see the Versions
-section of {{QUIC}}), and will be ignored when parsing these fields. On the
-other hand, the `Received Negotiation Version` list MUST be identical to the
-received Version Negotiation packet, so clients MUST NOT add or remove reserved
-version from that list.
+Clients and servers MAY both include versions following the pattern
+`0x?a?a?a?a` in their `Other Versions` list. Those versions are reserved to
+exercise version negotiation (see the Versions section of {{QUIC}}), and will
+be ignored when parsing these fields.
```

Comment:
These aren't ignored, it's just that they are never selected.

Reply:
Fixed